pymc/distributions/continuous.py

```diff
@@ -673,14 +673,14 @@ class TruncatedNormal(BoundedContinuous):
 
     Parameters
     ----------
-    mu: float
+    mu : tensor_like of float, optional
         Mean.
-    sigma: float
+    sigma : tensor_like of float, optional
         Standard deviation (sigma > 0).
-    lower: float (optional)
-        Left bound.
-    upper: float (optional)
-        Right bound.
+    lower : tensor_like of float, optional
+        Left bound. Defaults to 0.
```

Comment:
I think the defaults are wrong (same for upper). I'd bet the default is no bound or infinity or somehting of the sort

Reply:
should I remove the default values?

Reply:
I am unable to find out the default value for mu and sigma.
could you please guide me, how to identify these dafult value so that it can help me for other methords.

Reply:
First check the description in the docstring, sometimes the defaults are incorrectly explained in the extended summary section. If they are not you should read the `.dist` method and follow the variables defined there and/or function calls to see what the defaults are. Here are some examples, I hope they are illustrative and you can follow the pattern for the rest.

#### mu

https://github.com/pymc-devs/pymc/blob/4223b6e0600232b1074c1959337a1bea43b570b9/pymc/distributions/continuous.py#L720

it is defined straight away here, so it seems like it is not optional actually. @ricardoV94 is this intended? If the value in the function call were 0 instead of None then you would use the `default 0` instead of optional for example, however, if some function is called to get the default, it probably means you should use optional and explain the default (or combinations of defaults depending on other parameters) in the description

#### lower

https://github.com/pymc-devs/pymc/blob/4223b6e0600232b1074c1959337a1bea43b570b9/pymc/distributions/continuous.py#L724

the default in the function call is None, and when it is None, what is actually used is - infinity. When a value is given, that value is used. So lower should be `lower : tensor_like of float, default -numpy.inf` (and check in the readthedocs preview that it gets rendered correctly).

Reply:
Looks like the default of mu should indeed be zero, not None

Reply:
transform as a positional argument is outdated an should be removed from the `dist` signature